doc/page-features.h

```diff
@@ -52,6 +52,13 @@ lambda function:
     }
 \endcode
 
+\subsection async-sub4 Async with synchronous IPC proxy
+Asynchronous flag can be used with synchronous proxy, which changes behaviour of annotated method to be alwyas asynchronous.
```

Comment:
"Asynchronous flag" ? which one ? These sentences are confusing. The last 2 lines of this paragraph tell everything. Nothing else is needed, IMO .